README.md

```diff
@@ -1,8 +1,20 @@
 # multihash
 
+> hashpipe - pipe iff the hash matches
+
 Multihash is a protocol for differentiating outputs from various well-established cryptographic hash functions, addressing size + encoding considerations.
 
-It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See https://github.com/jbenet/random-ideas/issues/1 for a longer discussion.
+It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See [this GitHub thread](https://github.com/jbenet/random-ideas/issues/1) for a longer discussion.
```

Comment:
👎 i don't think it helps to hide what the link is.  the words  "this github thread" contain strictly less information than that link. and i do not think that link detracts from the paragraph